instruqt/avoiding-installation-pitfalls/02-checking-cluster-resources/assignment.md

```diff
@@ -75,35 +75,35 @@ for guidance on our preflights.
 
 Since we have both recommended and minimum values, we have thresholds
 for both warning and failure. Let's add the CPU check into our
-`harbor-preflights.yaml` manifest. Open up the "Manifest Editor" tab
+`slackernews-preflights.yaml` manifest. Open up the "Manifest Editor" tab
```

Comment:
my only note here is that the editor works against itself a little bit - the layout draws nice columns at 4-spaces but our yaml uses 2s, so editing in the manifest editor feels a little janky
![Screenshot 2024-04-23 at 13 22 34](https://github.com/replicatedhq/replicated-field-labs/assets/292598/17dbac62-5f35-46e8-9b41-d8164165e3bd)

Reply:
not sure if the editor can be configured for 2-space columns, but this is pretty minor